pkg/gather/clusterconfig/operators.go

```diff
@@ -130,82 +235,74 @@ func gatherClusterOperators(ctx context.Context, configClient configv1client.Con
 			})
 		}
 	}
+
+	return records
+}
+
+// gatherUnhealthyClusterOperator collects unhealthy cluster operator resources
+func gatherUnhealthyClusterOperator(ctx context.Context, items []configv1.ClusterOperator, coreClient corev1client.CoreV1Interface) ([]*corev1.Pod, []record.Record, int) {
+	var records []record.Record
+
 	namespaceEventsCollected := sets.NewString()
-	now := time.Now()
-	unhealthyPods := []*corev1.Pod{}
-	for _, item := range config.Items {
+	pods := []*corev1.Pod{}
+	totalContainers := 0
+
+	for _, item := range items {
 		if isHealthyOperator(&item) {
 			continue
 		}
+
 		for _, namespace := range namespacesForOperator(&item) {
-			pods, err := coreClient.Pods(namespace).List(ctx, metav1.ListOptions{})
+			pdl, err := coreClient.Pods(namespace).List(ctx, metav1.ListOptions{})
```

Comment:
There was a discussion some time ago about the use of C-style abbreviations in Go and why they're discouraged. I'm not really a supporter of the 10 words long Java-style naming conventions, but I believe that name should be clearly understandable even by someone who isn't familiar with the API. Names like `pdl`, `pds`, `tc` don't seem very descriptive. I'm mentioning this because you seem interested in best Go practices and these kinds of names are not generally considered a great practice in Go.

Reply:
I guess that it just depends on the size of the function body, and its context. 

If you have a function that has a huge body, it would be really hard to track down the variables without having a clean name. Also, in this case, it would be probably a good idea to split the function into multiple functions. But if you are using the variable in some small context, giving huge names is just extra verbosity to the code.

But, I agree, the names quoted above are not the most descriptive ones, and I'll change them.